### PR TITLE
Native Alerts instead red buttons

### PR DESCRIPTION
**Pull request by juanky201271**
* red fetch price button -> Alert.
* red confirm button in seed screen -> Alert.
* red confirm button in ufvk screen -> Alert.

### CODE REVIEW

**Comment by Oscar-Pepper** [comment]
wow looks really good! i think the alerts should match in style and maybe could be worded a bit better i will add comments in my review

File: app/translations/en.json [hunk 1]
```diff
@@ -225,21 +225,21 @@
         "",
         "I have saved \n the seed",
         "You really want \n to change your \n actual wallet",
-        "Are you sure \n 100% or more"
+        "Are you sure 100% or more"
```
Comment (Oscar-Pepper) [inline app/translations/en.json]: this should read:

Are you 100% sure?

File: app/translations/en.json [hunk 2]
```diff
@@ -225,21 +225,21 @@
...
       ],
       "server": [
         "",
         "I have saved \n the seed",
         "You really want \n to change your \n actual server",
-        "Are you sure \n 100% or more"
+        "Are you sure 100% or more"
```
Comment (Oscar-Pepper) [inline app/translations/en.json]: this should read:

Are you 100% sure?

File: app/translations/en.json [hunk 3]
```diff
@@ -225,21 +225,21 @@
...
       ],
       "view": ["I have saved \n the seed"],
       "restore": ["Restore Wallet"],
       "backup": [
         "",
         "I have saved \n the seed",
         "You really want \n to restore your \n backup wallet",
-        "Are you sure \n 100% or more"
+        "Are you sure 100% or more"
```
Comment (Oscar-Pepper) [inline app/translations/en.json]: this should read:

Are you 100% sure?

File: app/translations/en.json [hunk 4]
```diff
@@ -420,21 +424,21 @@
         "",
         "I have saved \n the Viewing Key",
         "You really want \n to change your \n actual wallet",
-        "Are you sure \n 100% or more"
+        "Are you sure 100% or more"
```
Comment (Oscar-Pepper) [inline app/translations/en.json]: this should read:

Are you 100% sure?

File: app/translations/en.json [hunk 5]
```diff
@@ -420,21 +424,21 @@
...
       ],
       "server": [
         "",
         "I have saved \n the Viewing Key",
         "You really want \n to change your \n actual server",
-        "Are you sure \n 100% or more"
+        "Are you sure 100% or more"
```
Comment (Oscar-Pepper) [inline app/translations/en.json]: this should read:

Are you 100% sure?

File: app/translations/en.json [hunk 6]
```diff
@@ -420,21 +424,21 @@
...
       ],
       "view": ["I have saved \n the Viewing Key"],
       "restore": ["Restore Wallet"],
       "backup": [
         "",
         "I have saved \n the Viewing Key",
         "You really want \n to restore your \n backup wallet",
-        "Are you sure \n 100% or more"
+        "Are you sure 100% or more"
```
Comment (Oscar-Pepper) [inline app/translations/en.json]: this should read:

Are you 100% sure?

**Comment by juanky201271** [comment]
> wow looks really good! i think the alerts should match in style and maybe could be worded a bit better i will add comments in my review

of course,,, I'm waiting for the new design to create a new custom component to replace the white and ugly alerts... https://github.com/zingolabs/zingo-mobile/issues/469
Thanks, I always need help with English Texts...

File: app/translations/en.json [hunk 7]
```diff
@@ -446,7 +450,8 @@
     "text-readonly": "This is your viewing key. Please write/copy it down carefully. Anyone who has a copy of the viewing key can observe all transactions to and from the associated wallet.",
     "change-warning": "IF YOU CHANGE TO ANOTHER WALLET, YOU WILL NO LONGER BE ABLE TO VIEW THIS WALLET WITHOUT THE VIEWING KEY.",
     "backup-warning": "IF YOU CHANGE TO YOUR BACKUP WALLET, YOU WILL NO LONGER BE ABLE TO VIEW THIS WALLET WITHOUT THE VIEWING KEY.",
-    "server-warning": "IF YOU CHANGE TO A SERVER WITH A DIFFERENT CHAIN, YOU WILL NO LONGER BE ABLE TO VIEW THIS WALLET WITHOUT THE VIEWING KEY"
+    "server-warning": "IF YOU CHANGE TO A SERVER WITH A DIFFERENT CHAIN, YOU WILL NO LONGER BE ABLE TO VIEW THIS WALLET WITHOUT THE VIEWING KEY",
```
Comment (Oscar-Pepper) [inline app/translations/en.json]: to match the other alert this should not be capitals and i think it looks better smaller in the alert:

"If you change to a server with a different chain, you will no longer be able to view this wallet without the viewing key"

File: app/translations/es.json [hunk 8]
```diff
@@ -446,7 +450,8 @@
     "text-readonly": "Esta es tu clave de visualización. Por favor apúntala/copiala cuidadosamente. Cualquiera que tenga una copia de esta clave puede ver las transacciones asociadas a este monedero.",
     "change-warning": "SI CAMBIAS A OTRO MONEDERO, YA NO PODRAS VER ESTE MONEDERO SIN LA CLAVE DE VISUALIZACION.",
     "backup-warning": "SI CAMBIAS A TU MONEDERO (COPIA DE SEGURIDAD), YA NO PODRAS VER ESTE MONEDERO SIN LA CLAVE DE VISUALIZACION.",
-    "server-warning": "SI CAMBIAS A UN SERVIDOR CON DIFERENTE TIPO DE CADENA, YA NO PODRAS VER ESTE MONEDERO SIN LA CLAVE DE VISUALIZACION."
+    "server-warning": "SI CAMBIAS A UN SERVIDOR CON DIFERENTE TIPO DE CADENA, YA NO PODRAS VER ESTE MONEDERO SIN LA CLAVE DE VISUALIZACION.",
```
Comment (Oscar-Pepper) [inline app/translations/es.json]: i think lower case letters looks better for this alert

**Review by Oscar-Pepper — CHANGES_REQUESTED** [review]
i think its looks best to have lower case in the alert to match the fetch USD alert...
can we change here also? https://github.com/zingolabs/zingo-mobile/pull/468/files#diff-c2ea5ebbb49266a16477e2cf76c9fa904d1faadbd317f92ccc6914874222b7cfR259

**Comment by zancas** [comment]
I assume @Oscar-Pepper will merge this one when he's satisfied.

File: app/translations/en.json [hunk 9]
```diff
@@ -259,14 +259,15 @@
     "tapcopy-birthday-message": "Copied Birthday to Clipboard",
     "birthday-readonly": "Wallet Birthday",
     "birthday-no-readonly": "Block height of first transaction. (It's OK, if you don't know) Value range between ",
-    "change-warning": "IF YOU CHANGE TO ANOTHER WALLET, YOU WILL NO LONGER HAVE ACCESS TO THIS WALLET WITHOUT THE SEED PHRASE.",
-    "backup-warning": "IF YOU CHANGE TO YOUR BACKUP WALLET, YOU WILL NO LONGER HAVE ACCESS TO THIS WALLET WITHOUT THE SEED PHRASE.",
-    "server-warning": "IF YOU CHANGE TO A SERVER WITH A DIFFERENT CHAIN, YOU WILL NO LONGER HAVE ACCESS TO THIS WALLET WITHOUT THE SEED PHRASE",
-    "mainnet-warning": "NO BACKUP OF THIS WALLET. ONLY AVAILABLE IN 'MAINNET'.",
+    "change-warning": "If you change to another wallet, you will no longer have access to this wallet without the see phrase.",
```
Comment (Oscar-Pepper) [inline app/translations/en.json]: All of these say 'see' instead of 'seed'

**Review by Oscar-Pepper — CHANGES_REQUESTED** [review]
Just need to change 'see' to 'seed' and good to go! See my comment

**Comment by juanky201271** [comment]
> Just need to change 'see' to 'seed' and good to go! See my comment

fixed!

File: app/translations/en.json [hunk 10]
```diff
@@ -225,21 +225,21 @@
         "",
         "Open New",
         "You really want \n to change your \n actual wallet",
-        "Are you sure \n 100% or more"
+        "Are you 100% sure?"
```
Comment (zancas) [inline app/translations/en.json]: I vote we change this to:   `Are you certain?`

File: app/translations/en.json [hunk 11]
```diff
@@ -299,14 +300,17 @@
...
     "sendall": "Send All",
     "sendall-message": "Your are about to send the entire amount in your wallet",
     "includeua": "Include your UA:",
     "confirm-memo": "Memo",
     "duplicate-nullifier-error": "Your funds are still confirming and probably syncing. You can wait until the funds are confirmed or you can try to send now with a lower amount. The Transaction build failed!",
     "dust-error" : "The amount that you are trying to send to a Transparent address is really tiny. The Transaction build failed!",
-    "needtoshield": "Have funds to shield"
+    "needtoshield": "Have funds to shield",
+    "fetchpricetitle": "Fetch USD Price",
+    "fetchpricebody": "Please confirm that you want to fetch the USD price from internet.",
```
Comment (zancas) [inline app/translations/en.json]: "from internet"  -->  "from the internet"

File: app/translations/en.json [hunk 12]
```diff
@@ -299,14 +300,17 @@
...
-    "nofetchprice": "No price",
+    "nofetchprice": "Fetch Price in USD",
     "sendall": "Send All",
     "sendall-message": "Your are about to send the entire amount in your wallet",
     "includeua": "Include your UA:",
     "confirm-memo": "Memo",
     "duplicate-nullifier-error": "Your funds are still confirming and probably syncing. You can wait until the funds are confirmed or you can try to send now with a lower amount. The Transaction build failed!",
     "dust-error" : "The amount that you are trying to send to a Transparent address is really tiny. The Transaction build failed!",
-    "needtoshield": "Have funds to shield"
+    "needtoshield": "Have funds to shield",
```
Comment (zancas) [inline app/translations/en.json]: "You have funds to shield."

**Review by zancas — CHANGES_REQUESTED** [review]
In the English version lets change `100%` to `certain`, I am not clear on how to change it in Spanish.

**Comment by juanky201271** [comment]
> In the English version lets change `100%` to `certain`, I am not clear on how to change it in Spanish.

in Spanish can be: `Estas seguro/a?`... I made all of your suggestions... @zancas

**Review by Oscar-Pepper — APPROVED** [review]

**Review by zancas — APPROVED** [review]

**Pull request merged**